sycl/plugins/cuda/CMakeLists.txt

```diff
@@ -101,6 +101,7 @@ add_sycl_plugin(cuda
     UnifiedRuntime-Headers
     UnifiedRuntimeCommon
     UnifiedMallocFramework
+    UnifiedRuntimeLoader
```

Comment:
Adapters shouldn't need to link the loader.

Reply:
It isn't linked in the Unified Runtime adapter, but in the PI plugin. I think that the PI plugin will require the loader to be linked in since it needs access to urLoaderInit/TearDown right?

Reply:
Right, I didn't notice that this is a PI plugin :) You are right that PI will need to link against loader to find these APIs, but doesn't it already do that [here](https://github.com/intel/llvm/pull/11155/files#diff-1bf2ed05c0065126d4e2150f0eda4dbd561e5c3625452e96356aadafa1312e6fL74)? I'm not sure, it's hard to tell due to complexity of these cmakes.

Reply:
You are right. Only the Unified-Runtime plugin should have the loader linked. The other PI plugins statically link in the implementation of the UR entry-points so don't require the use of `urLoaderInit` or `urLoaderTearDown`.